Sources/NIOPosix/Bootstrap.swift

```diff
@@ -3108,17 +3108,17 @@ public final class NIOPipeBootstrap {
     /// `ClientBootstrap.withConnectedSocket(descriptor:)` if it's a socket or
     /// `NIOPipeBootstrap.withInputOutputDescriptor` if it is not a socket.
     ///
-    /// - note: If this method returns a succeeded future, SwiftNIO will close `inputDescriptor` and `outputDescriptor`
+    /// - Note: If this method returns a succeeded future, SwiftNIO will close `inputDescriptor` and `outputDescriptor`
     ///         when the `Channel` becomes inactive. You _must not_ do any further operations `inputDescriptor` or
     ///         `outputDescriptor`, including `close`.
     ///         If this method returns a failed future, you still own the file descriptors and are responsible for
     ///         closing them.
     ///
-    /// - parameters:
-    ///     - inputDescriptor: The _Unix file descriptor_ for the input (ie. the read side).
-    ///     - outputDescriptor: The _Unix file descriptor_ for the output (ie. the write side).
-    /// - returns: an `EventLoopFuture<Channel>` to deliver the `Channel`.
-    public func withPipes(inputDescriptor: CInt, outputDescriptor: CInt) -> EventLoopFuture<Channel> {
+    /// - Parameters:
+    ///   - inputDescriptor: The _Unix file descriptor_ for the input (ie. the read side).
+    ///   - outputDescriptor: The _Unix file descriptor_ for the output (ie. the write side).
+    /// - Returns: an `EventLoopFuture<Channel>` to deliver the `Channel`.
+    public func takingOwnershipOfInputOutputDescriptor(inputDescriptor: CInt, outputDescriptor: CInt) -> EventLoopFuture<Channel> {
```

Comment:
Can we call this one `takingOwnershipOfDescriptors(input:output:)`?  There's a lot of repetition otherwise...

Reply:
Might make sense for the other one to be `takingOwnershipOfDescriptor(inputOutput:)` 🤷‍♂️ (sorry, I know I suggested `takingOwnershipOfInputOutputDescriptor(_:)` ...)